lib/date_time_parser/timezone_abbreviations.ex

```diff
@@ -0,0 +1,327 @@
+    "-0900" => "Etc/GMT+9",
+    "-1000" => "Etc/GMT+10",
+    "-1100" => "Etc/GMT+11"
+  }
+
+  @default_abbreviation_preferences %{
+    # Preferred GMT where possible, then USA, otherwise most populous.
+    "+01/+00" => "Africa/Casablanca",
+    "+02" => "Etc/GMT-2",
+    "+03" => "Etc/GMT-3",
+    "+04" => "Etc/GMT-4",
+    "+04/+05" => "Asia/Baku",
+    "+05" => "Etc/GMT-5",
+    "+06" => "Etc/GMT-6",
+    "+07" => "Etc/GMT-7",
+    "+08" => "Etc/GMT-8",
+    "+08/+09" => "Asia/Ulaanbaatar",
+    "+09" => "Etc/GMT-9",
+    "+10" => "Etc/GMT-10",
+    "+11" => "Etc/GMT-11",
+    "+11/+12" => "Pacific/Noumea",
+    "+12" => "Etc/GMT-12",
+    "+13" => "Etc/GMT-13",
+    "+13/+14" => "Pacific/Tongatapu",
+    "+14" => "Etc/GMT-14",
+    "-01" => "Etc/GMT+1",
+    "-01/+00" => "Atlantic/Azores",
+    "-02" => "Etc/GMT+2",
+    "-03" => "Etc/GMT+3",
+    "-03/-02" => "America/Buenos_Aires",
+    "-04" => "Etc/GMT+4",
+    "-04/-03" => "America/Santiago",
+    "-05" => "Etc/GMT+5",
+    "-05/-04" => "America/Bogota",
+    "-06/-05" => "Pacific/Galapagos",
+    "-08" => "Etc/GMT+8",
+    "-09" => "Etc/GMT+9",
+    "-10" => "Etc/GMT+10",
+    "-11" => "Etc/GMT+11",
+    # Australia
+    "AWST" => "Australia/Perth",
+    "ACWST" => "Australia/Eucla",
+    "ACT" => "Australia/Adelaide",
+    "ACDT" => "Australia/Adelaide",
+    "ACST" => "Australia/Adelaide",
+    "AET" => "Australia/Sydney",
+    "AEST" => "Australia/Sydney",
+    "AEDT" => "Australia/Sydney",
+    "LHDT" => "Australia/Lord_Howe",
+    # Americas
+    "ADT" => "America/Halifax",
+    "ALASKA" => "America/New_York",
+    "AKDT" => "America/Anchorage",
+    "AKST" => "America/Anchorage",
+    "AST" => "America/Puerto_Rico",
+    "CENTRAL" => "America/Chicago",
+    "CDT" => "America/Chicago",
+    "CST" => "America/Chicago",
+    "CT" => "America/Chicago",
+    "CWT" => "America/Belize",
+    "CPT" => "America/Belize",
+    "EASTERN" => "America/New_York",
+    "EDT" => "America/New_York",
+    "EST" => "America/New_York",
+    "ET" => "America/New_York",
+    "HAWAII" => "America/New_York",
+    "HST" => "Pacific/Honolulu",
+    "HAT" => "Pacific/Honolulu",
+    "HADT" => "Pacific/Honolulu",
+    "HAST" => "Pacific/Honolulu",
+    "MOUNTAIN" => "America/New_York",
+    "MDT" => "America/Denver",
+    "MST" => "America/Denver",
+    "MT" => "America/Denver",
+    "PACIFIC" => "America/New_York",
+    "PDT" => "America/Los_Angeles",
+    "PST" => "America/Los_Angeles",
+    "PT" => "America/Los_Angeles",
+    # Europe
+    "BT/BST" => "Europe/London",
+    "BMT/BST" => "Europe/London",
+    "BMT" => "Europe/London",
+    "BST" => "Europe/London",
+    "BT" => "Europe/London",
+    "CET/CEST" => "CET",
+    "CEST" => "CET",
+    "CET" => "CET",
+    "EET/EEST" => "EET",
+    "EEST" => "EET",
+    "EET" => "EET",
+    "WET/WEST" => "WET",
+    "WEST" => "WET",
+    "WET" => "WET",
+    "GMT" => "Etc/UTC",
+    "UTC" => "Etc/UTC",
+    "Z" => "Etc/UTC",
+    # Africa
```

Comment:
Feels like you're missing some African timezone abbreviations? https://www.timeanddate.com/time/zones/africa

Reply:
Ah ignore me then!